### PR TITLE
Launches the modal when adding geometries from the layer pane

### PR DESCRIPTION
**Pull request by javierarce**
This PR allows to use the Scratch modal to add geometries from the layer pane addFeature button. It also fixes a bug that prevented using the close button in that modal.

add

### CODE REVIEW

**Comment by javierarce** [comment]
@xavijam: quid pro quo ;)

File: lib/assets/javascripts/cartodb/common/dialogs/map/scratch_view.js [hunk 1]
```diff
@@ -33,7 +33,8 @@ module.exports = BaseDialog.extend({
   // implements cdb.ui.common.Dialog.prototype.render
   render_content: function() {
     return this._template({
-      name: this.table.get("name")
+      name: this.table.get("name"),
+      skipDisabled: this.options.skipDisabled || false
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/common/dialogs/map/scratch_view.js]: I'd create an ```options``` parameter with that value with false by default.
```js
...
options: {
   skipDisabled: false
}
...
```

Comment (xavijam) [inline lib/assets/javascripts/cartodb/common/dialogs/map/scratch_view.js]: I'd create an ```options``` parameter with that value with false by default.
```js
...
options: {
   skipDisabled: false
}
...
```

**Comment by xavijam** [comment]
Looks good to me!

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4266/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4269/))

**Comment by javierarce** [comment]
Danke!

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4275/))

**Pull request merged**